Modules/_datetimemodule.c

```diff
@@ -3007,13 +3013,13 @@ date_fromisoformat(PyObject *cls, PyObject *dtstr)
 static PyObject *
 date_fromisocalendar(PyObject *cls, PyObject *args, PyObject *kw)
 {
-    static char *keywords[] = {
+    static char *kwlist[] = {
```

Comment:
is there a reason all of the kwlist's everywhere cannot become static const?  (lemme guess.. our C API complains about the type?)